Assets/MRTK/Examples/Common/Scripts/DwellProfileWithDecay.cs

```diff
@@ -14,23 +14,11 @@ namespace Microsoft.MixedReality.Toolkit.Experimental.Dwell
     [Serializable]
     public class DwellProfileWithDecay : DwellProfile
     {
-        [Tooltip("Should the system allow for dwell to resume if the pointer exits the target briefly.")]
-        [SerializeField]
-        private bool allowDwellDecayOnCancel = false;
```

Comment:
Is this a feature we want to support, it just wasn't working? Or is it something else?

@ritijain

Reply:
I do not believe that Boolean is meaningful in its current state - the profile only works when it is true and also conceptually I do not see the reason setting it to false (i.e. if you do not want to resume why using dwell with decay in the first place?)

Reply:
@keveleigh - for gaze based dwell there are many things needed to stabilize the user's gaze and account for unintended errors... 
This property is expected to provide one of those stabilizations where - you intend to dwell on a button, but your gaze wanders off for a bit (couple ms), and so this property was meant to add some resiliency to the system to not cancel the dwell right away. This is sufficient for head gaze based dwell that Guides has, but not for eye gaze based dwell.

This iteration of work did not contain any support for eye gaze stabilization - but it may be worth keeping this around since the eye gaze stabilization logic that Sophie wrote in her examples was not ported  here.

Reply:
>it may be worth keeping this around since the eye gaze stabilization logic that Sophie wrote in her examples was not ported here.

@ritijain We do have _some_ smoothing for eye gaze from Sophie at https://github.com/microsoft/MixedRealityToolkit-Unity/blob/ab5c87a3c49478d05fd67d1ee8ed298a915c3f94/Assets/MRTK/Providers/WindowsMixedReality/XR2018/WindowsMixedRealityEyeGazeDataProvider.cs#L220-L290, but it might be out of date at this point

Reply:
@ritijain @keveleigh I understand the need for stabilization/not cancel the dwell just because the cursor goes off for a short amount of time, but the file in question is dwell with decay, which means it does not seem to make sense if we would keep that field around as in this case the field should _always_ be true. If that field is set to false then the "decay" no longer makes sense.